pkg/actors/reminders/reminders.go

```diff
@@ -205,7 +234,40 @@ func (r *reminders) DeleteReminder(ctx context.Context, req internal.DeleteRemin
 		<-r.evaluationChan
 	}()
 
-	return r.doDeleteReminder(ctx, req.ActorType, req.ActorID, req.Name)
+	config := retry.DefaultConfig()
+	config.Multiplier = 1.0
+	b := config.NewBackOffWithContext(ctx)
+	var etagErr *state.ETagError
+	var err error
+	err = retry.NotifyRecover(func() error {
+		innerErr := r.doDeleteReminder(ctx, req.ActorType, req.ActorID, req.Name)
+		if innerErr != nil {
+			if errors.As(innerErr, &etagErr) {
+				switch etagErr.Kind() {
+				case state.ETagMismatch:
+					// If the etag is mismatched, we can retry the operation.
+					return innerErr
+				default:
+					log.Errorf("error storing reminder: %w", innerErr)
+					err = innerErr
+					// we can't retry the operation.
+					return nil
+				}
+			}
+		}
+		log.Errorf("error storing reminder: %w", err)
+		err = innerErr
```

Comment:
Here too. Please, no side effects in goroutines. Use `NotifyRecoverWIthData`

Reply:
done